common/changes/@cadl-lang/compiler/symbolProviderUpdatedBranch_2022-07-23-03-49.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@cadl-lang/compiler",
+      "comment": "renamed function and constant variables",
+      "type": "minor"
```

Comment:
```suggestion
      "comment": "",
      "type": ""
```

Reply:
This just renames internal implementation details so doesn't need a changelog. You can just press Enter for None when you make a change like this.